### PR TITLE
Rename the module and resource to representor

### PR DESCRIPTION
**Pull request by kylef**
Let me know if there is anything I missed.

### CODE REVIEW

**Comment by smizell** [comment]
Should this bump version to 0.2 (if using semver)?

**Comment by honzajavorek** [comment]
This is backwards incompatible change, so if major version is greater than 0, semver requires to bump the major one. However, as far as I can see (only scanned the spec), there are no strict rules for what happens until the major version is 1:

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Hypermedia Resource - Python
-----------------------------
+Hypermedia Representor - Python
```
Comment (fosrias) [inline README.md]: Any reason we have left Hypermedia here vs. just representor?

File: setup.py [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
       author='Stephen Mizell',
       author_email='smizell@gmail.com',
-      url='https://github.com/the-hypermedia-project/hypermedia-resource-python/',
-      packages=['hypermedia_resource',
-                'hypermedia_resource.adapters',
-                'hypermedia_resource.contrib'],
+      url='https://github.com/the-hypermedia-project/representor-python/',
+      packages=['representor-python',
+                'representor-python.adapters',
+                'representor-python.contrib'],
```
Comment (honzajavorek) [inline setup.py]: shouldn't these be just `representor`?

Comment (kylef) [inline setup.py]: Completely right, I don't know what I was thinking on that day.

**Comment by kylef** [comment]
This pull-request has been updated to address feedback.

**Pull request merged**